Dockerfile

```diff
@@ -0,0 +1,39 @@
+
```

Comment:
nit, if it's relevant: Are there any dependencies or instructions to add to run this in the README?